internal/pkg/cli/delete/static_site_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package delete
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/aws/copilot-cli/internal/pkg/cli/delete/mocks"
```

Comment:
I'm totally OK with just keeping what you have 👍 

My personal preference nowadays is just to implement the test double myself and make it clear what [type of](https://samuel.karp.dev/blog/2023/02/flexible-test-doubles-in-go/) double it is.

examples:
1. https://github.com/aws/copilot-cli/blob/836be87d5ff9b2912bbf5ba97626252dac62b21b/internal/pkg/template/overrides_test.go#L62-L76
2. https://github.com/aws/copilot-cli/blob/836be87d5ff9b2912bbf5ba97626252dac62b21b/internal/pkg/template/templatetest/templatetest.go#L14

Reply:
i like it! i incorporated this feedback and stuff found https://google.github.io/styleguide/go/best-practices#test-double-packages-and-types - let me know what you think!